spec/motions-spec.coffee

```diff
@@ -633,9 +632,10 @@ describe "Motions", ->
           keydown('-')
 
         it "selects to the first character of the previous line (directly above)", ->
-          expect(editor.getText()).toBe "abcdefg\n"
+          # commented out because vim does not put a newline but vim-mode does
+          #expect(editor.getText()).toBe "abcdefg"
```

Comment:
I pulled these commits out of a hairier patchset fixing https://github.com/atom/vim-mode/pull/386 and accidentally let this line come along for the ride.

Technically, this assertion is wrong -- vim doesn't put the newline.  But that's a fix for another day.